webapp/src/workspace.ts

```diff
@@ -79,13 +80,30 @@ export function setupWorkspace(id: string) {
         case "idb":
             impl = indexedDBWorkspace.provider;
             break;
+        case "cloud":
+            impl = cloudWorkspace.provider;
         case "browser":
         default:
             impl = browserworkspace.provider
             break;
     }
 }
 
+// TODO @darzu: needed?
```

Comment:
Not needed. This was needed when we would wholesale switch to cloud workspace after authentication.